src/lib.rs

```diff
@@ -1023,8 +980,7 @@ unsafe fn close_internal(c_out: &mut *mut Rav1dContext, flush: c_int) {
             freep(&mut (*f).frame_thread.cbi as *mut *mut CodedBlockInfo as *mut c_void);
         }
         if (*c).n_tc > 1 as c_uint {
-            pthread_mutex_destroy(&mut (*f).task_thread.pending_tasks.lock);
-            pthread_cond_destroy(&mut (*f).task_thread.cond);
+            mem::take(&mut (*f).task_thread.pending_tasks); // TODO: remove when context is owned
```

Comment:
```suggestion
            let _ = mem::take(&mut (*f).task_thread.pending_tasks); // TODO: remove when context is owned
```

I've been using `let _ = mem::take` as a searchable idiom for explicitly dropping a `Default`able field in a type that's not yet correctly dropped.